lib/transformations/utils.js

```diff
@@ -83,7 +83,7 @@ function findPluginsRootNodes(j, node) {
 function createProperty(j, key, value) {
 	return j.property(
 		'init',
-		j.literal(key),
+		j.identifier(key),
```

Comment:
Yeah but as far as I can see we don't use anything but identifiers in our code so I'd create a separate GHI, replace it to `j.identifier` and add a comment to the GHI for now. Since this style will break linting of lots of code I guess.